libpimeval/src/pimSim.h

```diff
@@ -133,6 +133,10 @@ class pimSim
   ~pimSim();
   pimSim(const pimSim&) = delete;
   pimSim operator=(const pimSim&) = delete;
+  void init();
```

Comment:
Can we just keep one init function? The simConfig parameter can be empty string by default and act exactly like this

Reply:
You can keep this or change the other init file to take file name as param

Reply:
Fixed!